src/plugins/eqht-css/_base.scss

```diff
@@ -0,0 +1,25 @@
+.wb-eqht-grd{
+	align-items: stretch;
+	display: flex;
+	flex-wrap: wrap;
+	height: 100%;
+
+	>[class*="col-"]{
+		display:flex;
+		flex-direction: column;
+		flex-grow:1;
+	}
+}
+.wb-eqht-grd .hght-inhrt{
+	display: flex;
+	flex: 1 1 auto;
+	flex-direction: column;
+
+	> *{
+		display: flex;
+		flex-direction: column;
+	}
```

Comment:
This will need to be removed

Reply:
Seems needed to me.  How can I verify this?  What CSS will replace this if removed?

Reply:
Yes correct, the plugin will add the "hght-inhrt" class making the above 2 properties selected with custom selector redundant.

-removed